appinventor/blocklyeditor/src/blocks/components.js

```diff
@@ -170,6 +170,13 @@ Blockly.Blocks.component_event = {
     if (this.instanceName == oldname) {
       this.instanceName = newname;
       this.componentDropDown.setValue(this.instanceName);
+      if (this.isCollapsed())
+      {
+        var COLLAPSED_INPUT_NAME = '_TEMP_COLLAPSED_INPUT';     
```

Comment:
We tend to use variables IN ALL CAPS as constants. I would recommend you choose different names for this variables.